src/private/meetingAudio.ts

```diff
@@ -0,0 +1,45 @@
+import { ensureInitialized, sendMessageRequestToParent } from '../internal/internalAPIs';
+import { GlobalVars } from '../internal/globalVars';
+
+export namespace meetingAudio {
+  export interface IToggleResponse {
```

Comment:
Instead of doing this can you please follow the pattern of the new media APIs like captureImage, getMedia, etc. You can see them here:
https://github.com/OfficeDev/microsoft-teams-library-js/blob/master/src/public/media.ts

Instead of the developer passing an object with a successCallback and failureCallback we now pass a single callback which takes two parameters - error and result. If we hit any issues then an error is returned and null result. Otherwise, a null error and valid result are returned.

Reply:
Thank you for the details. @ydogandjiev I've made updates.